### ISSUE TITLE
intent_classifier_sklearn throwing error and I cant find any example to use it.

### ISSUE THREAD

**Issue by avnkailash** [RasaHQ/rasa_nlu#1480]
**Rasa NLU version**:
0.13.7

**Operating system** (windows, osx, ...):
osx

**Content of model configuration file**:
```yml
#language: en
#pipeline: tensorflow_embedding

language: "en"

pipeline:
- name: "nlp_spacy"
  model: "en"
  case_sensitive: false
- name: "tokenizer_spacy"
- name: "ner_spacy"
  case_sensitive: false
- name: "ner_duckling_http"
  url: "http://0.0.0.0:8443"
  dimensions:
  - "NUMBER"
- name: "ner_synonyms"
- name: "ner_crf"
  BILOU_flag: true
  features:
    # features for word before token
    - ["low", "title", "upper", "pos", "pos2"]
    # features of token itself
    - ["bias", "low", "upper", "title", "digit", "pos", "pos2", "pattern"]
#    , "word3", "word2", "upper", "title", "digit", "pos", "pos2", "pattern"]
    # features for word after the token we want to tag
    - ["low", "title", "upper", "pos", "pos2"]
  max_iterations: 50
  L1_c: 1
  L2_c: 1e-3
- name: "intent_classifier_sklearn"
```

**Issue**:
```
Traceback (most recent call last):
  File "/usr/local/Cellar/python/3.6.5/Frameworks/Python.framework/Versions/3.6/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/local/Cellar/python/3.6.5/Frameworks/Python.framework/Versions/3.6/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/Users/vanantha/Desktop/Projects/yac/venv/lib/python3.6/site-packages/rasa_nlu/train.py", line 184, in <module>
    num_threads=cmdline_args.num_threads)
  File "/Users/vanantha/Desktop/Projects/yac/venv/lib/python3.6/site-packages/rasa_nlu/train.py", line 154, in do_train
    interpreter = trainer.train(training_data, **kwargs)
  File "/Users/vanantha/Desktop/Projects/yac/venv/lib/python3.6/site-packages/rasa_nlu/model.py", line 186, in train
    components.validate_arguments(self.pipeline, context)
  File "/Users/vanantha/Desktop/Projects/yac/venv/lib/python3.6/site-packages/rasa_nlu/components.py", line 86, in validate_arguments
    "".format(component.name, r))
Exception: Failed to validate at component 'intent_classifier_sklearn'. Missing property: 'text_features'
```

I am unable to figure out why the pipeline is failing at the intent_classifier_sklearn stage. I have referre so many examples on the internet but still no where I can find anything related to this text_features issue.

Pleas help me in getting unblocked and get this pipeline to working state.

**Comment by akelad** [comment]
closing because it's a duplicate of #1481 (probably caused by yesterdays github issues)
